promptsource/templates/glue/mrpc/templates.yaml

```diff
@@ -1,35 +1,49 @@
-dataset: glue
-subset: mrpc
-templates:
-  3b88d2c4-0aeb-4c6d-9ccc-653a388250a5: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 3b88d2c4-0aeb-4c6d-9ccc-653a388250a5
-    jinja: '{% if label == 1 %}
-
+"dataset": |-
+  glue
+"subset": |-
+  mrpc
+"templates":
+  "3b88d2c4-0aeb-4c6d-9ccc-653a388250a5": !Template
+    "answer_choices": !!null |-
+      null
+    "answer_choices_key": !!null |-
+      null
+    "id": |-
+      3b88d2c4-0aeb-4c6d-9ccc-653a388250a5
+    "jinja": |-
+      {% if label == 1 %}
       Paraphrase the following sentence: {{sentence1}}
 
       |||
 
       {{sentence2}}
 
-      {% endif %}'
-    metadata: !TemplateMetadata
-      choices_in_prompt: false
-      metrics:
-      - BLEU
-      - ROUGE
-      original_task: false
-    name: generate_paraphrase
-    reference: ''
-  44c2e6d9-facf-4959-8400-38e0eb8dd3a8: !Template
-    answer_choices:
-    - 'no'
-    - 'yes'
-    answer_choices_key: null
-    id: 44c2e6d9-facf-4959-8400-38e0eb8dd3a8
-    jinja: 'I want to know whether the following two sentences mean the same thing.
-
+      {% endif %}
```

Comment:
The block scalar's literal style `|(-)` doesn't need single or double quotes, and newlines will guarantee WYSIWYG.

If we want one the jinja to act in this manner, then we will need to modify more places like this: https://stackoverflow.com/a/20863889